public/js/common/modal.js

```diff
@@ -218,7 +218,7 @@ function createG(nodeId) {
     g.attr('class', 'tooltip-group')
         .css('cursor', 'pointer')
         .click(function () {
-            openModal(nodeId);
+            openCourseModal(nodeId);
```

Comment:
Eh, I don't think this needs to be a helper. The facebook one just does it inline, which is fine here too.